### PR TITLE
add description on spotPrice

### PR DESCRIPTION
**Pull request by yinan-liu**

### CODE REVIEW

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc [hunk 1]
```diff
@@ -181,12 +180,13 @@ TIP: If all the nodes of an AWS instance are removed, the instance will be termi
...
     
     -   *minCores:* The minimum required amount of virtual cores for each AWS instance that needs to be created.
-    
+
 +
 WARNING: If the combination between RAM and CORES does not match any existing AWS instance type, then the closest to the specified parameters will be selected.
-+
-    
 
+    -   *spotPrice:* Amazon EC2 Spot instances allow you to bid on spare Amazon EC2 computing capacity. Since Spot instances are often available at a discount compared to On-Demand pricing. More information available documentation on https://aws.amazon.com/ec2/spot/[AWS Spot]
```
Comment (marcocast) [inline src/docs/admin/references/NodeSourceInfrastructureReference.adoc]: Can you add "The maximum price that you are willing to pay per hour per instance (your bid price). "

**Comment by marcocast** [comment]
👍

**Comment by bamedro** [comment]
(...) More information available documentation on https://aws.amazon.com/ec2/spot/[AWS Spot] (...)

--> More information available on https://.... No ?

**Comment by yinan-liu** [comment]
Yes, OK

**Pull request merged**